extensions/edge_extensions/reference/utils.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Utilities
+order: 16
+---
+
+# Utilities
+
+The `utils` object is a "variable" accessible inside your extension's library modules. It provides utilities specific to the Launch runtime and is always available as the 3rd parameter when your module is executed.
```

Comment:
"The `utils` object is a "variable" accessible inside your extension's library modules." -> Consider something like "The `utils` object is provided to your extension's library modules." I'm assuming `utils` is not a free variable.